### PR TITLE
Use autocomplete for selecting taxon's parent

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | yes |
| Related tickets | closes #7750 |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php [hunk 1]
```diff
@@ -116,35 +117,7 @@ public function selectMainTaxon(TaxonInterface $taxon)
...
-        });
-
-        $mainTaxonItemElement = $mainTaxonElement->find('css', sprintf('div.item:contains("%s")', $taxon->getName()));
-
-        $mainTaxonItemElement->click();
-
-        $this->getDocument()->waitFor(5, function () use ($isVisibleScript) {
-            return !$this->getDriver()->evaluateScript($isVisibleScript);
-        });
+        AutocompleteHelper::chooseValue($this->getSession(), $mainTaxonElement, $taxon->getName());
```
Comment (Arminek) [inline src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php]: 👍

**Comment by pjedrzejewski** [comment]
Thanks Kamil! Did you discuss the tree functionality with @Arminek?

**Comment by pamil** [comment]
Making it a tree would not require too much work, custom form type like `TaxonTreeChoiceType` with its own theme should do the work.

**Comment by pjedrzejewski** [comment]
@pamil Let's do this then. WDYT?

**Pull request merged**